tests/BusinessModelTest.php

```diff
@@ -222,12 +222,9 @@ public function testSetTitle()
     {
         $m = new Model();
         $m->addField('name');
-        $m->set('foo');
+        $m->set('name', 'foo');
```

Comment:
I think we can/should introduce `$model->setTitle()` method which could cover this functionality. Especially because we already have `$model->getTitle()`, so having also setTitle() is only logical, right?

Reply:
No, title is not that special, this is bad, we should not use title in data at all (only support title field for ui)

title is non-sense from data perspective, title it often also not unique and so on...

Reply:
Title is also very useful in data not just UI. And I think it's nice to have shortcut method
`$model->setTitle('foo');` instead of `$model->set($model->title_field, 'foo');`
Same as `$model->getTitle()` instead of longer `$model->get($model->title_field);` (plus fallback to $model->id)